#goal
int32 detection_mode
---
#result
int32 result
---
#feedback
int32 number_of_faces